.github/workflows/ci-nx.yml

```diff
@@ -78,6 +78,28 @@ jobs:
         run: |
           echo ::set-output name=affected::"$(npx nx print-affected --type=app | jq -c .projects)"
 
+  fs-scan:
+    name: vulnerability-scanning
+    needs: ci
+    runs-on: ubuntu-20.04
+    uses: amplication/amplication/.github/workflows/template-vulnerability-scan.yaml@master
+    with:
+      scan-type: 'fs'
+      format: 'sarif'
+      exit-code: 0
+      publish: true
+
+  config-scan:
+    name: vulnerability-scanning
+    needs: ci
+    runs-on: ubuntu-20.04
+    uses: amplication/amplication/.github/workflows/template-vulnerability-scan.yaml@master
+    with:
+      scan-type: 'config'
+      format: 'sarif'
+      exit-code: 0
+      publish: true
```

Comment:
Actually, let's setup a new workflow called `security.yml` that will use the same trigger events of `ci-nx.yml` 

This will allow us to quickly visualise the failures due to security scanning from the code-related failures.
 
@levivannoort do you agree?

Reply:
I do agree, the way I set it up was with the thought in mind that we wanted to integrate it in the current pipeline and maybe fail if there are dependencies with vulnerabilities that the person that opened a pull-request first needs to fix.

Reply:
I see and it makes sense.